app/src/main/java/com/example/task1/HomeFragment.kt

```diff
@@ -0,0 +1,82 @@
+package com.example.task1
+
+import android.content.Intent
+import android.os.Bundle
+import androidx.fragment.app.Fragment
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+import android.widget.TextView
+import android.widget.Toast
+import androidx.databinding.DataBindingUtil
+import androidx.recyclerview.widget.LinearLayoutManager
+import androidx.recyclerview.widget.RecyclerView
+import com.example.task1.databinding.HomeFragmentAllMoviesBinding
+import org.w3c.dom.Text
+import retrofit2.Call
+import retrofit2.Callback
+
+private const val ARG_PARAM1 = "param1"
+private const val ARG_PARAM2 = "param2"
+
+class HomeFragment : Fragment(),MovieClicked {
+
+    private lateinit var myCustomAdapter: CustomeAdapter
+    private var moviesList = mutableListOf<Movie>()
+
+    private lateinit var binding : HomeFragmentAllMoviesBinding
+
+    private var param1: String? = null
+    private var param2: String? = null
+
```

Comment:
rename the params to be meaningful names, a name that describes what the param holds